volume.go

```diff
@@ -295,3 +295,33 @@ func (v *Volume) SetVolumeSize(sizeInGB string) error {
 		http.MethodPost, path, payload, nil)
 	return err
 }
+
+// FindVolumeByIopsOrBandwidth : Get the list of volume with specified IOPS or Bandwidth
+func (c *Client) FindVolumeByIopsOrBandwidth(LimitIops int, LimitBwInMbps int) ([]*types.Volume, error) {
+	volList, err := c.GetVolume("", "", "", "", false)
+	if err != nil {
+		return nil, err
+	}
+	var volumesNew []*types.Volume
+	//negative value of LimitIops or LimitBwInMbps indicates that
+	//don't populates volumes based on that value
+	for _, vol := range volList {
+		for _, sdc := range vol.MappedSdcInfo {
+			if LimitIops >= 0 && LimitBwInMbps >= 0 {
+				if sdc.LimitIops == LimitIops && sdc.LimitBwInMbps == LimitBwInMbps {
+					volumesNew = append(volumesNew, vol)
+				}
+			} else if LimitIops >= 0 && LimitBwInMbps < 0 {
```

Comment:
If LimitIops or LimitBwInMbps are <0 then it should be handled as a error. Why are we considering them in the conditional. Can you please help me in understanding the logic.

Reply:
I've removed my previous changes as it was not required .